eos-toolbox-dev-depends

```diff
@@ -0,0 +1,74 @@
+# The eos-toolbox-dev package is intended to be included in the eos-toolbox
+# container images as a way to provide the base development packages needed
+# for common development.
+#
+# Packages listed here should be commonly used, of high value for
+# development, and expected to be relevant to a whole group of Endless
+# developers.
+#
+# Some packages included in the eos-dev package (e.g. gnome dev packages)
+# are not included here on purpose given they increase the container image
+# size quite substantially. Developers that need the extra packages can
+# still install them manually using apt-get from an eos-toolbox container.
+
+acl
+apt-file
+apt-transport-https
+apt-utils
+autoconf
+automake
+bash-completion
+buildah
+build-essential
+bzip2
+ca-certificates
+ccache
+curl
+dbus
+debootstrap
+devscripts
+diffutils
+endless-ca-cert
+findutils
+flatpak-xdg-utils
+gdb
+git
+gnupg
+hostname
+htop
+iftop
+iotop
+less
+libcap2-bin
+libnss-myhostname
+locales-all
+lsof
+make
+man-db
+manpages
+meson
+net-tools
+obs-build
+openssh-client
+osc
+passwd
+patchutils
+pbuilder
+pep8
+podman
+procps
+python
+python3
+quilt
+rsync
+strace
+sudo
+time
+traceroute
+unzip
+util-linux
+vim
```

Comment:
Well, maybe. I actually do use Vim, but I have weaned myself onto the Flathub package of it. Is this needed in the countainer?

Reply:
hehe, I may be biased here cause I only use vim as editor :). I would like to have a better editor than the default "vi" in the image but am Ok also to let the developer install whatever editor they want if you think that is an issue.